example/deployment/local-docker/example-docker-runner/src/main/java/uk/gov/gchq/palisade/example/RestDockerExample.java

```diff
@@ -70,16 +72,56 @@ public void run() throws Exception {
 
         final ExampleSimpleClient client = new ExampleSimpleClient(palisade);
 
+        final User alice = new User()
```

Comment:
Can you refactor this into an ExampleUsers class as part of the example-model rather than defining this in 3 different places. Can you also uncomment the auths so that the auths are set for each user (even though they are not used by the current policies)
Can you also use the Roles enum class (found in example-model, common class), e.g. Role.HR.name()